package.json

```diff
@@ -19,10 +19,10 @@
     "clean": "rimraf lib dist es",
     "typecheck:flow": "flow check && [[ $(flow check) == 'Found 0 errors' ]]",
     "lint": "eslint src",
-    "test:only": "jest",
+    "test:only": "npm run build:umd && jest",
```

Comment:
Why we need to build umd to test:only?
I usually use this `test:only` to do fast tests...

Reply:
Because puppeteer tests use umd build

Reply:
Oh, got it.